src/main/app/src/assets/i18n/en.json

```
@@ -759,6 +759,8 @@
   "msg.error.besluit.publication.date.missing": "Missing decision publication date",
   "msg.error.besluit.response.date.missing": "Missing decision response date",
   "msg.error.besluit.response.date.invalid": "Response date for decision should not be before publication date",
+  "msg.error.case.has.open.subcases": "This case has open subcases and can not be closed",
+  "msg.error.case.has.locked.information.objects": "this case has locked information objects and can not be closed",
```

```suggestion
  "msg.error.case.has.locked.information.objects": "This case has locked documents and cannot be closed",
```

As far as I know we don't use the term 'information objects' in the UI.